cves/kernel/CVE-2021-41073.yml

```diff
@@ -221,7 +229,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: io_uring
   note:
```

Comment:
I'd explain why you think in the note. Also, io_uring.c is located in the fs directory so it might be part of the fs subsystem as well